modules/viz/include/opencv2/viz/viz3d.hpp

```diff
@@ -247,7 +247,8 @@ namespace cv
             @param specularColor The specular color of the light
              */
             void addLight(Vec3d position, Vec3d focalPoint = Vec3d(0, 0, 0), Color color = Color::white(),
-                          Color diffuseColor = Color::white(), Color ambientColor = Color::black(), Color specularColor = Color::white());
+                          Color diffuseColor = Color::white(), Color ambientColor = Color::black(),
+                          Color specularColor = Color::white()) const;
```

Comment:
`const` should not be appended at the end of the method which makes `this` constant and forbids the following statement

https://github.com/opencv/opencv/blob/e0c888acf78ea1ecfe03c6c7d55e041fc96f1750/modules/viz/src/vizimpl.cpp#L221

The parameters should be instead `const`, that is,
```.cpp
const Vec3d &position, const Vec3d &focalPoint, .........
```

Do not forget to change the following (and its header file)
https://github.com/opencv/opencv/blob/e0c888acf78ea1ecfe03c6c7d55e041fc96f1750/modules/viz/src/vizimpl.cpp#L206